package.json

```diff
@@ -1,84 +1,96 @@
-    "npm": ">=7.7.6"
-  },
-  "files": [
-    "/dist"
-  ],
-  "main": "dist/src/plugin.js",
-  "scripts": {
-    "lint:fix": "eslint --fix .",
-    "lint": "eslint .",
-    "type": "tsc --noEmit",
-    "build": "etsc",
-    "watch": "nodemon",
-    "test": "jest"
-  },
-  "types": "lib/index.d.ts"
+    "name": "serverless-lift",
+    "license": "MIT",
+    "homepage": "https://github.com/getlift/lift",
+    "repository": "https://github.com/getlift/lift",
+    "description": "Lift",
+    "dependencies": {
+        "@aws-cdk/aws-apigatewayv2": "1.107",
+        "@aws-cdk/aws-apigatewayv2-integrations": "1.107",
+        "@aws-cdk/aws-certificatemanager": "1.107",
+        "@aws-cdk/aws-cloudfront": "1.107.0",
+        "@aws-cdk/aws-cloudfront-origins": "1.107",
+        "@aws-cdk/aws-cloudwatch": "1.107.0",
+        "@aws-cdk/aws-events": "1.107.0",
+        "@aws-cdk/aws-iam": "1.107.0",
+        "@aws-cdk/aws-lambda": "1.107.0",
+        "@aws-cdk/aws-s3": "1.107.0",
+        "@aws-cdk/aws-sns": "1.107.0",
+        "@aws-cdk/aws-sqs": "1.107.0",
+        "@aws-cdk/core": "1.107.0",
+        "aws-sdk": "^2.737.0",
+        "chalk": "^4.1.0",
+        "change-case": "^4.1.1",
+        "cidr-split": "^0.1.2",
+        "js-yaml": "^3.14.1",
+        "lodash": "^4.17.21",
+        "log-symbols": "^4.0.0",
+        "mime-types": "^2.1.30",
+        "node-notifier": "^8.0.0",
+        "ora": "^5.0.0",
+        "pascal-case": "^3.1.2",
+        "tslib": "^1.13.0"
+    },
+    "devDependencies": {
+        "@serverless/test": "^7.11.0",
+        "@serverless/typescript": "^2.35.0",
+        "@types/chai": "^4.2.12",
+        "@types/jest": "^26.0.22",
+        "@types/js-yaml": "^3.12.5",
+        "@types/json-schema": "^7.0.7",
+        "@types/mime-types": "^2.1.0",
+        "@types/node": "^10.17.28",
+        "@types/node-notifier": "^8.0.0",
+        "@types/sinon": "^10.0.1",
+        "@typescript-eslint/eslint-plugin": "^4.21.0",
+        "@typescript-eslint/parser": "^4.21.0",
+        "chai": "^4.2.0",
+        "esbuild-node-tsc": "^1.4.0",
+        "eslint": "^7.23.0",
+        "eslint-config-prettier": "^8.1.0",
+        "eslint-plugin-import": "^2.22.1",
+        "eslint-plugin-prettier": "^3.3.1",
+        "globby": "^10.0.2",
+        "husky": "^6.0.0",
+        "jest": "^26.6.3",
+        "json-schema-to-ts": "^1.6.1",
+        "lint-staged": "^11.0.0",
+        "nodemon": "^2.0.7",
+        "prettier": "^2.2.1",
+        "serverless": "^2.35.0",
+        "sinon": "^9.2.4",
+        "stdout-stderr": "^0.1.13",
+        "ts-jest": "^26.5.4",
+        "typescript": "^4.2.4"
+    },
+    "peerDependencies": {
+        "serverless": "^2.35.0"
+    },
+    "engines": {
+        "node": ">=15.14.0",
+        "npm": ">=7.7.6"
+    },
+    "files": [
+        "/dist"
+    ],
+    "main": "dist/src/plugin.js",
+    "scripts": {
+        "lint:fix": "eslint --fix .",
+        "lint": "eslint .",
+        "check-format": "prettier --check .",
```

Comment:
I'm not sure a new command is needed, we already have `npm run lint`.

In case I'm wrong, it might be worth updating https://github.com/getlift/lift/blob/master/.github/CONTRIBUTING.md

Reply:
`eslint` will only format js and ts files, for all other files, we need to use `prettier`.
It's a better practice to separate the two, since `eslint` is doing static analysis and `prettier` is only formatting.
That being said, I could change the existing commands to:

```json
    "lint:fix": "prettier --write . && eslint --fix .",
    "lint": "prettier --check . && eslint .",
```

What do you think ?